cookbooks/travis_ci_ruby/attributes/default.rb

```diff
@@ -1,11 +1,11 @@
-override['rvm']['default'] = '2.2.5'
+override['rvm']['default'] = '2.2.6'
 override['rvm']['rubies'] = %w(
   1.9.2
   1.9.3
   2.0.0
   2.1.10
-  2.2.5
-  2.3.1
```

Comment:
Are you intending to remove these Rubies from being preinstalled? This will slow down builds for everyone still on 2.3.1 when 2.3.3 only came out this week.